Src/C64.cpp

```diff
@@ -793,10 +793,18 @@ void C64::VBlank(bool draw_frame)
 #endif
 }
 
+#ifdef RS90
+extern short shiftstate;
+#endif
+
 /*  Poll joystick port, return CIA mask */
 uint8 C64::poll_joystick(int port)
 {
+#ifdef RS90
```

Comment:
I can if I assume that we switch to my type of layout, I can do that. Otherwise it makes no sense as this flag shows that DPad is used elsewhere.
I can merge my version of layout (If you like it) with generic code and switch unused in RS90 stuff (kb and mouse) there off

Reply:
Or we can make a general define that will enable layout and other things not exclusive to RS90, but for the all small devices out there